### ISSUE TITLE
Миграции

### ISSUE THREAD

**Issue by SuperBilge** [soovorow/yii2-product-features-module#1]
Установил модуль. Подключил в конфиге, сайт на меня ругается, кажется не хватает миграций для полноценной работы модуля

**Comment by soovorow** [comment]
@SuperBilge Верно, не хватает миграций и getDb() переопределены под не под дефолтную базу. Выдергивал модуль из под живого проджекта. Постараюсь поправить в ближайшее время
